package.json

```diff
@@ -24,6 +25,7 @@
     "xtend": "^4.0.0"
   },
   "devDependencies": {
-    "from": "^0.1.3"
+    "from": "^0.1.3",
+    "q": "^1.4.1"
```

Comment:
The reason is that child-process-promise relies on Q and this implementation detail leaks.